src/config.txt.sample

```diff
@@ -42,11 +42,13 @@ password=admin
 database=postgres
 port=5432
 
-[EXPORT_CONFIG]
-path=exports/
+[API_CONFIG]
+export_path=exports/ # used to store export path
 api_host=http://127.0.0.1
 api_port=8000
-max_area=100000
+max_area=100000 # max area to support for rawdata input 
+use_connection_pooling=true # to use connection pooling , remove this block if you don't need connection pooling
```

Comment:
`s/this block/this line`

We can make this false in the example to offer a sane default. Let's not encourage removal of lines. Rather, they should set it to false if they don't need it. Explicit is better than implicit.